public/docs/ts/latest/guide/router.jade

```diff
@@ -868,27 +869,72 @@ a#nav-to-list
 
 +makeExcerpt('app/heroes/hero-detail.component.1.ts (excerpt)', 'gotoHeroes', '')
 
-h3#merge-hero-routes Import hero module into root NgModule
+h3#route-animation Adding animations to the route component
 :marked
-  Our heroes feature is ready, but application doesn't know about our heroes module yet.
-  We'll need to import it into the root NgModule we defined in `app.module.ts`.
+  Our heroes feature module is almost complete, but what is a feature without some smooth transitions?
+  We already know that Angular supports [animations](../guide/animations.html) and we want to take
+  advantage of them by adding some animation to our *Hero Detail* component.
+
+  First, we'll start by importing our animation functions that build our animation triggers,
+  control state and manage transitions between states. We'll use these functions to add transitions
+  to our route component as it moves between states our application view. We'll also import the
+  `HostBinding` decorator for binding to our route component.
+
++makeExcerpt('app/heroes/hero-detail.component.ts (animation imports)', 'route-animation-imports')
+
+:marked
+  Next, we'll use a **host binding** for route animations named *@routeAnimation*. There is nothing special
+  about the choice of the binding name, but since we are controlling route animation, we'll go with it.
+  The binding value is set to `true` because we only care about the `*` and `void` states which are
+  [entering and leaving](../guide/animations.html#example-entering-and-leaving) animation states.
+
+  We'll also add some display and positioning bindings for styling.
+
++makeExcerpt('app/heroes/hero-detail.component.ts (route animation binding)', 'route-animation-host-binding')
+
+:marked
+  Now we can build our animation trigger, which we'll call *routeAnimation* to match our binding we previously
+  setup. We'll use the **wildcard state** that matches any animation state our route component is in, along with
+  two *transitions*. One transition will animate our component entering the application view (`void => *`), while the other
+  controls our component leaving the application view (`* => void`).
+
+  We could add different transitions to different route components depending on our needs, but we'll just animate our
+  *Hero Detail* component for this milestone.
+
+.l-sub-section
+  :marked
+    We'll cover transitions between **route paths** in a future update.
```

Comment:
We could do that. But this entire section leaves the reader with some natural questions about how to do multi route animations/transitions without copying and pasting this code to every component. And when to use this technique vs using CSS animations for this.